source/IntegrationTests/DocumentAsserters/NotifyWholesaleServicesDocumentAsserter.cs

```diff
@@ -90,14 +90,12 @@ public static async Task AssertCorrectDocumentAsync(DocumentFormat documentForma
         var isTotalMonthlyAmount = assertionInput.Points.First().Price is null;
         if (!isTotalMonthlyAmount)
```

Comment:
```suggestion
        if (!isTotalOrMonthlyAmount)
```
Isn't this correct? It should not be present with either total or montly sums

Reply:
Maybe this change is incorrect. But it is hard to determine if this change is only for the Monthly amount messages or for both Total and Monthly amount messages

Reply:
this should only happen for TotalMonthlyAmount and not MonthlyAmount because MonthlyAmount has more fields on the point to validate